docs/changelog/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert/): properly update value of variable `$activeAt` in rules annotation during replay mode. Before, `$activeAt` could have provided incorrect values during replay.
 * BUGFIX: [MetricsQL](https://docs.victoriametrics.com/metricsql/): properly handle `c1 AND c2` and `c1 OR c1` queries for constants `c1` and `c2`. Previously such queries could return unexpected results. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6637).
 * BUGFIX: all VictoriaMetrics components: increase default value of `-loggerMaxArgLen` cmd-line flag from 1000 to 5000. This should improve visibility on errors produced by very long queries.
+* BUGFIX: [stream aggregation](https://docs.victoriametrics.com/stream-aggregation/): support having multiple labels per `remoteWrite.url` using `-remoteWrite.streamAggr.dropInputLabels`. Before, labels were set to all the `remoteWrite.url`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6780) for the details.
```

Comment:
@Haleygo should we also mention somewhere that this change breaks previous logic?

Reply:
In https://github.com/VictoriaMetrics/VictoriaMetrics/commit/57183c9b612a06685be2bc07d879743c75d2be82,  the bugfix description was removed from `update note`. 
This pull request should probably follow the same rule, or both of them should be mentioned in `update note`. I'd like to confirm with @hagen1778